notes/notes.md

```diff
@@ -1,22 +1,22 @@
 # Notes about Raspberry Pi Experiment
 
-This page contains the main scenario for running the experiment
+This page describes the main scenario for running the experiment
 
 
 ## 1 - Boot2Raspberry
 
-First of all, we have to dump a working OS to an SD-Card and boot on it.
```

Comment:
In this case, the exact word to use is "dump" (this is a block-level copy, while install have some implicit semantic) :)